openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -108,7 +108,9 @@ def _add_toggle_instance_details(self, toggle, toggle_instance):
         toggle['class'] = toggle_instance.__class__.__name__
         toggle['module'] = toggle_instance.module_name
```

Comment:
Since we also wouldn't want to include this if module is None, let's create an outer if that protects this and simplify the inner code.